packages/msw-addon/src/initializeWorker.js

```diff
@@ -0,0 +1,66 @@
+import { setupWorker } from 'msw'
+import { setupServer } from 'msw/node'
+
+export class Worker {
+  constructor() {
+    this.worker = setupWorker()
+  }
+
+  start(options) {
+    this.worker.start(options)
+  }
+
+  use(...handlers) {
```

Comment:
What is the benefit of abstracting the worker/server API like this?
Calling `setupWorker`/`setupServer` returns an identical API that works differently under the hood. I think we're not benefiting from this abstraction, as we'd have to map each and every new method through it. I'd suggest removing it, unless there's a really good reason to have it.

Reply:
Thanks for reviewing! Well, it was a way to address the comment you added [here](https://github.com/mswjs/msw-storybook-addon/pull/27#discussion_r630547184). Essentially, the addon would expose a single interface for the user (which could be properly typed if we migrate to ts) and that interface would take care of mapping the correct methods. If we don't do that, then the `worker` instance exported either via `setupWorker` or `setupServer` will be confusing for the end user.

Reply:
Got it. I think you can achieve this uniformity without abstracting the API into a class. See my suggestion in the refactored `initialize` function.